assets/js/components/HostDetails/HostDetails.jsx

```diff
@@ -24,6 +25,16 @@ import {
   sapInstancesTableConfiguration,
 } from './tableConfigs';
 
+const semver = require('semver');
+
+export const agentVersionWarnings = (agentVersion) => {
```

Comment:
maybe `agentVersionWarningsCheck` or something more descriptive?

Reply:
Changed to `agentVersionWarning`, as by now we only have one warning message